e2e/packages/contracts/mud.config.ts

```diff
@@ -50,4 +50,11 @@ export default defineWorld({
       key: [],
     },
   },
+  modules: [
+    {
+      name: "DelegationModule",
```

Comment:
We should call this something more specific, maybe `DelegationWithSignatureModule`

Reply:
Agreed, trying to find a balance bc `registerDelegationWithSignature` is so long to write

Reply:
I think it'd be ok since it's just temporary until we'll include it in the `WorldRegistrationModule` by default. We could even name it in some way that makes it explicit that this is temporary.